{{cookiecutter.github_repository_name}}/etc/helm/staging/values.yaml

```diff
@@ -1,22 +1,24 @@
 deployment:
   command:
-    - gunicorn
-    - "config.wsgi:application"
+  - gunicorn
+  - config.wsgi:application
   args:
-    config: "file:config/gunicorn.py"
+    config: file:config/gunicorn.py
   replicaCount: 1
-  resources: null
+  resources:
 envFromSecret:
   enabled: true
   secretName: {{ cookiecutter.app_name }}
 image:
   pullPolicy: IfNotPresent
   repository: {{ cookiecutter.docker_image_staging }}
   tag: latest
+imagePullSecrets:
```

Comment:
This is not needed for services running on GKE. We might not be able to use `helm_chart_version: 0.2.3`, till iv fixed the imagePullSecrets being optional ?

Reply:
Yes, I thought so which is why I only added it to the staging values for now and not the production values.